### PR TITLE
Remove broken links from Showcase page and correct spelling

### LINKED ISSUE #2567: Showcase Links: Update/Remove Broken Links
In line with compliance for TTS reporting, we are expediting removal and update of broken links on Showcase page. 
Syncing recent Siteimprove report with audit doc from November: 

Siteimprove report: https://docs.google.com/spreadsheets/d/12gIwMXlH_KKJL4DBmrG5r0EbTKp7eBYAIy0ELVVsn-U/edit#gid=678722873
Showcase page link report: https://docs.google.com/spreadsheets/d/1kp8za2t48RRnFRrLadYhn5Ou5gM6JcD1m8gEOD-q3CU/edit#gid=0

Related to: https://github.com/uswds/uswds-site/issues/2346 and https://github.com/uswds/uswds-site/issues/2407

Will be closed by : #2586

### PR DESCRIPTION
**Pull request by finekatie**
# Summary

Remove broken links that have no alternate for redirects. Correct misspelling that Siteimprove noted. Will improve QA on site.

## Related issue #2567 
Also related to: #2346 and #2407

Closes #2567

## Preview link

[Showcase page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-showcase-linkremoval/documentation/showcase/)

## Problem statement

Broken links and misspelling on this page contributes to drop in QA.

## Solution

Removing old broken links and correcting spelling will improve content relevancy and QA.

## Changes

https://datacenters.cio.gov/	remove 
https://feedback.usa.gov/	remove
https://itmodernization.cio.gov/	remove 
https://www.move.mil/	remove
http://vsfs.state.gov/	remove

## Testing and review

Will need to have someone review to make sure links and copy are gone and that spelling is correct. 

Check that changelog is implemented correctly and the content is correct.

### CODE REVIEW

**Review by mejiaj — COMMENTED** [review]
@finekatie only found references for `move.mil` on a blog post. The rest don't appear anywhere.

File: pages/documentation/showcase.md [hunk 1]
```diff
@@ -93,7 +91,6 @@ If your project is currently using USWDS and you would like to see it included i
 - [Manufacturing USA](https://www.manufacturing.gov/)
 - [Medicaid](https://www.medicaid.gov/)
 - [Method Cards - 18F](https://methods.18f.gov/)
-- [Move.mil — Official DOD Moving Portal](https://www.move.mil/)
 - [MyMedicare](https://www.mymedicare.gov/)
 - [NASA General Coordinates Network (GCN)](https://gcn.nasa.gov)
 - [NASA Glenn Research Center](https://www1.grc.nasa.gov)
```
Comment (sarah-sch) [inline pages/documentation/showcase.md:96]: @finekatie - while we're updating the page can we also correct the spelling of "Satellite" in "[National Environmental Sattelite Data and Information Service (NOAA)](https://www.nesdis.noaa.gov/)" (This issue was also flagged in the Siteimprove report.)

**Review by sarah-sch — CHANGES_REQUESTED** [review]

File: pages/documentation/showcase.md (hunk 1 above)
Comment (finekatie) [reply pages/documentation/showcase.md:96]: Hey there, @sarah-sch , I can, but since this isn't a broken link, do I need to update the Issue, etc.?

Comment (sarah-sch) [reply pages/documentation/showcase.md:96]: Discussed with @finekatie on 4/2 and updated issue

File: pages/documentation/showcase.md [hunk 2]
```diff
@@ -93,7 +91,6 @@ If your project is currently using USWDS and you would like to see it included i
 - [Manufacturing USA](https://www.manufacturing.gov/)
 - [Medicaid](https://www.medicaid.gov/)
 - [Method Cards - 18F](https://methods.18f.gov/)
-- [Move.mil — Official DOD Moving Portal](https://www.move.mil/)
```
[the remark this answers is not in the document]
Comment (finekatie) [reply pages/documentation/showcase.md:96]: Thank you James. We'll address that page and work to be done in another issue and PR when we go to clean up old blog pages.

File: pages/documentation/showcase.md (hunk 1 above)
Comment (mejiaj) [reply pages/documentation/showcase.md:96]: Confirming this is fixed in 64f6c0c0.

**Review by mejiaj — APPROVED** [review]

**Review by sarah-sch — APPROVED** [review]
Looks good!

**Comment by mejiaj** [comment]
@finekatie I've updated the labels in the original issue. Also confirming there aren't any `test` errors.

**Comment by mahoneycm** [comment]
It looks like we've added changelogs for removed links before. Should we add a changelog to the showcase page?

Example: [About research page](https://designsystem.digital.gov/about/research/#latest-updates)

**Comment by finekatie** [comment]
> It looks like we've added changelogs for removed links before. Should we add a changelog to the showcase page? There currently isn't a changelog section for this page. Happy to create if we decide we want one!
> 
> Example: [About research page](https://designsystem.digital.gov/about/research/#latest-updates)

Hi Charlie! We are talking about this as I write this. In favor of a changelog. Thank you!

File: _data/changelogs/docs-showcase.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title: Showcase
+type: documentation
+changelogURL:
+items:
+  - date: NNNN-NN-NN
+    summary: Remove broken links that have no alternate for redirects. 
+    summaryAdditional: Correct misspelling that Siteimprove noted.
+    affectsGuidance: true
+    githubPr: 2586
+    githubRepo: uswds-site
```
Comment (mahoneycm) [inline _data/changelogs/docs-showcase.yml:10]: ## Created changelog

After discussing with the team, we decided to add a changelog and keep track of larger changes such as addition and removal of links.

File: _data/changelogs/docs-showcase.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+title: Showcase
+type: documentation
+changelogURL:
+items:
+  - date: NNNN-NN-NN
```
Comment (thisisdano) [inline _data/changelogs/docs-showcase.yml:5]:
```suggestion
  - date: 2024-04-26
```

Add current date

**Review by thisisdano — APPROVED** [review]

**Comment by thisisdano** [comment]
@finekatie I'm on it. We needed to update another broken link

**Comment by thisisdano** [comment]
Merging over unrelated pa11y crash

**Pull request merged**